grants.d/grants.yml

```diff
@@ -1006,6 +1006,12 @@
     - project:
         feature: is-trunk
 
+- grant:
+    - queue:get-artifact:{artifact_prefix}/*
+  to:
+    - project:
+        artifact_prefix: "*"
```

Comment:
I think this grants the scope to untrusted pull requests, which is probably not desired. We might not even want pushes to have this in some cases :/

Reply:
I think as part of the feature (or repo config), we need to somehow specify what jobs are allowed to access private artifacts, as this will differ from repo to repo.